test_runner/pg_clients/swift/PostgresClientKitExample/Sources/PostgresClientKitExample/main.swift

```diff
@@ -3,21 +3,22 @@ import Foundation
 import PostgresClientKit
 
 do {
+    let env = ProcessInfo.processInfo.environment
+
     var configuration = PostgresClientKit.ConnectionConfiguration()
+    let host = env["NEON_HOST"] ?? ""
+    configuration.host = env["NEON_HOST"] ?? host
```

Comment:
```swift
configuration.host = host
```

you already made this magic with `env[NEON_HOST] ?? ""`

Reply:
Good catch, thanks!